scripts/rollup/results.json

```diff
@@ -33,24 +33,24 @@
       "gzip": 29313
     },
     "react.development.js (NODE_DEV)": {
-      "size": 71914,
-      "gzip": 18276
+      "size": 72595,
+      "gzip": 18499
     },
     "react.production.min.js (NODE_PROD)": {
       "size": 9195,
       "gzip": 3614
     },
     "React-dev.js (FB_DEV)": {
-      "size": 73423,
-      "gzip": 18751
+      "size": 74104,
+      "gzip": 18972
     },
     "React-prod.js (FB_PROD)": {
       "size": 36836,
       "gzip": 9248
     },
     "ReactDOMStack-dev.js (FB_DEV)": {
-      "size": 496986,
-      "gzip": 118763
+      "size": 497667,
+      "gzip": 118999
```

Comment:
I think every time. This came up in code review before, on https://github.com/facebook/react/pull/9420 and @trueadm said: 

> For now, whenever you update something that will be merged into master (it's always good to merge master into your branch and run this after). It gives us all some indications on possible mistakes in terms of bundle size regressions. We'll improve how this works in the coming weeks too.

![screen shot 2017-05-24 at 7 52 18 am](https://cloud.githubusercontent.com/assets/1114467/26409674/149d7ba8-4056-11e7-8cf8-417b4869b8ea.png)